packages/java-edition/src/mcfunction/completer/argument.ts

```diff
@@ -56,9 +59,17 @@ import {
 } from '../node/index.js'
 import type { ArgumentTreeNode } from '../tree/index.js'
 
+function getItemStackFormat(ctx: CompleterContext): 'old' | 'new' {
+	const release = ctx.config.env.gameVersion as ReleaseVersion | undefined
```

Comment:
What's the difference between this and `ctx.project['loadedVersion]`?

Reply:
I think `ctx.project['loadedVersion]` should be used here? `ctx.config.env.gameVersion` is the raw config value the user provides (it could be something weird like `'auto'` and `'latest snapshot'`) while `ctx.project['loadedVersion]` is an actual game version we parsed